libs/components/src/chip-select/chip-select.component.html

```diff
@@ -52,10 +52,12 @@
         type="button"
         bitMenuItem
         (click)="viewOption(parent, $event)"
-        [title]="parent.label ? ('backTo' | i18n: parent.label) : ('back' | i18n)"
+        [title]="
+          parent.label ? ('backTo' | i18n: parent.label) : ('backTo' | i18n: placeholderText)
+        "
```

Comment:
You can clean these up slightly by doing ```suggestion
        [title]="'backTo' | i18n: (parent.label ? parent.label : placeholderText)"
```